### PR TITLE
finished week-4

### PR DESCRIPTION
**Pull request by MichaelReyes1**
# Submitting Week 4 Assessment

### ✅ Completed Files

Select the following as confirmation that the files in this repository are complete.

- [ X] Behavioral Questions
- [ X] Career Readiness
- [ X] Coding Challenges
- [ X] Interview Questions
- [ X] Ruby Challenges

### ✋ Reminder

Do NOT merge your assessment!

### 📝 Feedback

If you would like specific feedback please leave details here:

### CODE REVIEW

File: behavioral-questions.md [hunk 1]
```diff
@@ -4,16 +4,24 @@ Answer the following questions. Use the Nugget + STAR method for each answer.
 
 1. Why did you choose a career in software development?
 
-- Nugget:
-- Situation:
-- Task:
-- Action:
-- Result:
+- Nugget: I want to secure better career options for myself.
```
Comment (elysemontano) [inline behavioral-questions.md:7]: When responding to this question, consider some of the attributes that software development has that interest you.  Interviewers like to hear deep answers that show that your interests align with theirs.  For instance, if you enjoy problem solving, or puzzles, or expanding on how you love learning could make this answer stand out.

File: code-challenges.test.js [hunk 2]
```diff
@@ -15,24 +15,83 @@
...
 
 // b) Create the function that makes the test pass.
 
+const removeFirstAndShuffle = function(arr) {
+    if (arr.length > 1) {
+      // If the array has more than one item
+      const firstItemRemoved = arr.slice(1)
+       // Remove the first item
+  
+      // Shuffle the items randomly
```
Comment (elysemontano) [inline code-challenges.test.js:43]: Nice work in this section!  You could expand a little more on your pseudocode specifically for line 44 and talk about how this is randomly shuffling the items.

File: ruby-challenges.rb [hunk 3]
```diff
@@ -3,34 +3,81 @@
...
 palindrome_test_case3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
+puts check_palindrome(palindrome_test_case3)
+
+# Pseudo code:
 
-# Pseudo code:
\ No newline at end of file
+# create function check_palindrome that takes in a string
+# using .reverse to compare to original string
```
Comment (elysemontano) [inline ruby-challenges.rb:82]: It looks like you are not comparing to the original string in your code.  You may also need to consider the casing as well to get the correct output.

**Review by elysemontano — APPROVED** [review]
Excellent work on this assessment Michael!  This assessment displays an Advanced Beginner level of skill acquisition.  Some things to focus on to really make yourself level up and stand out would be to expand and get deeper on interview and behavioral questions.  Keep up the great work!

File: behavioral-questions.md (hunk 1 above)
Comment (MichaelReyes1) [reply behavioral-questions.md:7]: copy i will def add that

**Pull request merged**